src/bot/index.ts

```diff
@@ -187,7 +177,50 @@ bot.on("Room.timeline", (event: mSDK.MatrixEvent) => {
   }
 });
 
-export const startBot = () => {
-  validateConfig("BOT");
-  bot.startClient({ initialSyncLimit: 0 }).catch((e) => logger.error(e));
-};
+export async function startBot(): Promise<void> {
+  if (!accessToken) return;
```

Comment:
@aramikm This is the one line I added in here to disable the bot if we don't have an access token